buildbot/configure.py

```diff
@@ -198,6 +202,10 @@ def do_configure(args):
             "-DSYCL_LIBCXX_INCLUDE_PATH={}".format(args.libcxx_include),
             "-DSYCL_LIBCXX_LIBRARY_PATH={}".format(args.libcxx_library)])
 
+    if 'openmp' in llvm_in_tree_projects:
+      cmake_cmd.extend([
+            "-DOPENMP_ENABLE_LIBOMPTARGET=OFF"])
+
```

Comment:
This deserves a comment, IMO.

Reply:
Hi @aelovikov-intel,

It means that we are going to compile OpenMP without any GPU-offloading support. I will leave a few comment lines in the script

Reply:
Why is that something specific to this script rather than your particular usage scenario?